shaman/controllers/nodes/__init__.py

```diff
@@ -42,6 +43,14 @@ def index(self):
             resp[node.host] = node.__json__()
         return resp
 
+    @secure(basic_auth)
+    @expose(method='GET', content_type="text/plain")
+    def next(self):
+        next_node = get_next_node()
+        if not next_node:
+            abort(404, "There are no healthy chacra nodes available.")
+        return next_node.host
```

Comment:
do we want to include scheme? e.g. https or http ? that would make us assume though :(